src/main/java/github/scarsz/discordsrv/DiscordSRV.java

```diff
@@ -1768,9 +1765,13 @@ public void broadcastMessageToMinecraftServer(String channel, Component message,
                 .map(hook -> (ChatHook) hook)
                 .findAny().orElse(null);
 
+        DiscordGuildMessagePreBroadcastEvent preBroadcastEvent = api.callEvent(new DiscordGuildMessagePreBroadcastEvent
+                (channel, message, PlayerUtil.getOnlinePlayers()));
+        message = preBroadcastEvent.getMessage();
+        channel = preBroadcastEvent.getChannel();
         if (chatHook == null || channel == null) {
-            if (channel != null && !channel.equalsIgnoreCase("global")) return; // don't send messages for non-global channels with no plugin hooks
-            MessageUtil.sendMessage(PlayerUtil.getOnlinePlayers(), message);
+            if (channel != null && !channel.equals("global")) return; // don't send messages for non-global channels with no plugin hooks
```

Comment:
Why the `String#equalsIgnoreCase` to `String#equals`

Reply:
Honestly, I don't remember doing that or why I did it... will switch it back.